### PR TITLE
Propose Swift OpenAPI Generator

### PR DESCRIPTION
**Pull request by czechboy0**
## Summary

This proposes the endorsement of https://github.com/apple/swift-openapi-generator by the SSWG, because it fills a missing gap in supporting client and server code generation from OpenAPI documents in the Swift ecosystem.

We are proposing it for Sandbox.

The primary authors are: @czechboy0 and @simonjbeaumont

## Test Plan

N/A

### CODE REVIEW

**Comment by ktoso** [comment]
I think this is ready to be run -- checking who would like to review manage it 👍

File: proposals/NNNN-swift-openapi-generator.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Swift OpenAPI Generator
+
+* Proposal: [SSWG-NNNN](NNNN-swift-openapi-generator.md)
+* Authors: [Honza Dvorsky](https://github.com/czechboy0)
+* Review Manager: TBD
```
Comment (Joannis) [inline proposals/NNNN-swift-openapi-generator.md]: You can add me as a review manager.

Comment (czechboy0) [reply proposals/NNNN-swift-openapi-generator.md]: Done 🙏

**Review by Joannis — COMMENTED** [review]
These are not applicable yet, right? Can we omit them from this PR?

**Comment by czechboy0** [comment]
> These are not applicable yet, right? Can we omit them from this PR?

What are you referring to? 🙂

File: proposals/NNNN-swift-openapi-generator.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+* Review Manager: [Joannis Orlandos](https://github.com/joannis)
+* Status: **Implemented**
+* Implementation: [apple/swift-openapi-generator](https://github.com/apple/swift-openapi-generator)
+* Forum Threads: [Pitch](https://forums.swift.org/t/pitch-sswg-incubation-swift-openapi-generator/66584), [Discussion](https://forums.swift.org/), [Review](https://forums.swift.org/)
+
+*During the review process, add the following fields as needed:*
+
+* Decision Notes: [Rationale](https://forums.swift.org/), [Additional Commentary](https://forums.swift.org/)
+* Previous Revision(s): [1](https://github.com/swift-server/sswg/blob/...commit-ID.../proposals/NNNN-filename.md)
+* Previous Proposal(s): [SSWG-XXXX](XXXX-filename.md)
```
Comment (Joannis) [inline proposals/NNNN-swift-openapi-generator.md]: @czechboy0 that was supposed to be tagged to these lines

Comment (czechboy0) [reply proposals/NNNN-swift-openapi-generator.md]: Removed: https://github.com/swift-server/sswg/pull/86/commits/cf500b30b85419ea4bc838d93325a07f8a4007d2

**Review by Joannis — APPROVED** [review]
Looks good. Merging and creating the forum thread

**Pull request merged**